src/cli/run.rs

```diff
@@ -167,6 +184,7 @@ fn command_not_found<'p>(project: &'p Project, explicit_environment: Option<Envi
             project
                 .environments()
                 .into_iter()
+                .filter(|env| verify_current_platform_has_required_virtual_packages(env).is_ok())
```

Comment:
Maybe instead of filtering we should show this information to the user.

Reply:
Could also do that in a follow-up PR.

Reply:
Yeah lets get this in, was already looking at some info logging as normal warnings would be to verbose in my tests.